pkg/webhook/board_update.go

```diff
@@ -60,16 +78,91 @@ func (h *boardUpdate) handleIssuesEvent(event *github.IssuesEvent, gh *github.Cl
 	logger.Debug("Issue Action: " + *event.Action)
 
 	eventKey := messageType + "_" + *event.Action
+
+	// post processing (from previous event cycle)
+	// takes precedence, the event will no be processed further
+	if "issues_closed" == eventKey {
+		_, issue_scheduled := postProcessing[number]
+
+		if issue_scheduled {
+			delete(postProcessing, number)
+			go postProcess(event, gh, config, *doneColumn, logger)
+			return nil
+		}
+	} else if "issues_reopened" == eventKey && event.GetIssue().GetLocked() {
+		// move
+		err := moveIssueOnBoard(config, number, *doneColumn, logger)
+
+		// update progress/* label
+		changeProgressLabel(gh, event.Repo, *event.Issue, doneColumn.name)
+
+		if err != nil {
+			logger.Error("Post processing failed: Cannot move issue")
+		}
+	}
+
+	// cleanup post processing markers, but skip actions
+	if event.GetIssue().GetLocked() {
+		gh.Issues.Unlock(context.Background(), event.Repo.Owner.GetLogin(), event.Repo.GetName(), *event.Issue.Number)
+		logger.Debug("Unlock and ignore event for: " + number)
```

Comment:
Why is the event ignored ? hasnt it be already treated above (e.g. for "issues_reopened") ?

Reply:
some events are triggered by post processing. we need to ignore a followup event otherwise we get unintended side effects. the locked state is used to indicate that post processing is going on and while that happens all events triggered by it should be ignored.

Reply:
Yeah, but you will enter the conditional also when you have handled the 'reopen' event (which is only performed when locked). So this event is not ignored for this event

but its a nit-pick anyway, so don't bother too much with this.